### ISSUE TITLE
Ctrl+C to stop iex / mix doesn't work on Windows

### ISSUE THREAD

**Issue by imetallica** [elixir-lang/elixir#7910]
### Environment

* Elixir & Erlang/OTP versions (elixir --version): 
```
Erlang/OTP 21 [erts-10.0.1] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1]

Elixir 1.6.6 (compiled with OTP 19)
```
* Operating system: `Windows 10 version 1803`

### Current behavior
When running `iex.bat -S mix` or `iex.bat` we cannot stop the process just doing `CTRL+C`. Looks like it ignores the command whatsoever.

### Expected behavior
Stop the application correctly.

**Comment by radarjones** [comment]
I believe this an upstream problem in Erlang/OTP see 
http://erlang.org/pipermail/erlang-questions/2018-July/095969.html

**Comment by imetallica** [comment]
@radarjones `CTRL+C` works with raw `erl` command. But doesn't with any of the Elixir family of commands, `iex` nor `mix`.

**Comment by josevalim** [comment]
There is also a thread in the Elixir forum: https://elixirforum.com/t/unable-to-kill-the-vm-with-ctrl-c-erlang-21-0-1-and-elixir-1-6-6-on-windows/14872

@imetallica can you try `erl -noshell` and see if Ctrl + C works? That's pretty much what Mix does. If that doesn't work, then that's the root cause we should report to Erlang/OTP.

**Comment by michalmuskala** [comment]
It seems this is a bug in the new IO polling implementation on Windows.

http://erlang.org/pipermail/erlang-questions/2018-July/095969.html

**Comment by imetallica** [comment]
@josevalim I'll give a shot later because I downgraded Erlang to 20, so I can work.

**Comment by josevalim** [comment]
@imetallica it has already been determined to be an Erlang bug, so no worries.

**Comment by josevalim** [comment]
This is fixed in OTP 21.1 and the online installed has been updated to provide 21.1 as a download option.
